docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -55,6 +55,19 @@ We have added 4 new features to the flamegraph visualization:
 
 {{< video-embed src="/media/docs/grafana/panels-visualizations/screen-recording-grafana-10.1-flamegraph-whatsnew.mp4" >}}
 
+### Visualizations and Widgets split
+
+<!-- Alexa Vargas -->
+
+_Experimental in all editions of Grafana._
+
+This experimental feature introduces a clear distinction between two different categories of panel plugin types: visualization panels that consume a data source and a new type called _widgets_ that don't require a data source. 
+
+Now, you can easily add widgets like Text, News, and Annotation list without the need to select a data source. The plugins list and library panels are filtered based on whether you've selected a widget or visualization, providing a streamlined editing experience. If you want to try this new feature, you can enable the feature flag, `vizAndWidgetSplit`.
```

Comment:
Adding enablement info for Cloud and using standard language for the feature toggle

```suggestion
Now, you can easily add widgets like Text, News, and Annotation list without the need to select a data source. The plugins list and library panels are filtered based on whether you've selected a widget or visualization, providing a streamlined editing experience. 

To try out the visualizations and widgets split, enable the `vizAndWidgetSplit` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
```

Reply:
@imatwawana I added the suggestion manually, I was resolving a conflict and could not add the commit from here. Can you take a last look? 🙈

Reply:
Looks good!